.github/workflows/preview-build.yml

```diff
@@ -64,6 +64,7 @@ jobs:
         uses: tj-actions/changed-files@2f7c5bfce28377bc069a65ba478de0a74aa0ca32 # v46.0.1
         with:
           files: ${{ inputs.path-pattern != '' && inputs.path-pattern || '**' }}
+          files_separator: ','
```

Comment:
Where is this needed?

IMO the newline separator is more readable

```yaml
        with:
          # Avoid using single or double quotes for multiline patterns
          files: |
            **.md
            docs/**.md
```

Source: https://github.com/tj-actions/changed-files?tab=readme-ov-file#using-local-git-directory-